src/test/kotlin/sh/hsp/techtree/graphviz/DSLConverterTest.kt

```diff
@@ -0,0 +1,49 @@
+package sh.hsp.techtree.graphviz
+
+import io.kotest.matchers.shouldBe
+import org.junit.jupiter.api.Test
+import sh.hsp.techtree.TreeModel
+import sh.hsp.techtree.TreeNode
+
+class DSLConverterTest {
+
+    @Test
+    fun givenTreeModelWithSingleNodeShouldRegisterSingleNodeInCorrectGrapvizDSL() {
+        val model = TreeModel(listOf(TreeNode("node")))
+
+        val converter = DSLConverter()
+
+        val result = converter.convert(model)
+
+        "digraph { node }".shouldBe(result)
+    }
+
+    @Test
+    fun givenTreeModelWithMultipleSingularNodesShouldRegisterAllSingularNodesInCorrectGrapvizDSL() {
+        val model = TreeModel(listOf(TreeNode("nodeA"), TreeNode("nodeB"), TreeNode("nodeC")))
+
+        val converter = DSLConverter()
+
+        val result = converter.convert(model)
+
+        "digraph { nodeA nodeB nodeC }".shouldBe(result)
```

Comment:
@oneacik `shouldBe()` to najbliższe temu o czym gadaliśmy, bo wypluwa takie coś w razie niepowodzenia: 

```
Expected :"digraph { nodeA nodeB nodeC }"
Actual   :"digraph { nodeA nodeBs nodeC }"
<Click to see difference>
```

Reply:
Potem może poszukam czegoś lepszego

Reply:
KKK